tests/components/sonos/test_media_player.py

```diff
@@ -683,6 +686,147 @@ async def test_select_source_error(
     assert "Could not find a Sonos favorite" in str(sve.value)
 
 
+async def test_shuffle_set(
+    hass: HomeAssistant,
+    soco: MockSoCo,
+    async_autosetup_sonos,
+) -> None:
+    """Test the set shuffle method."""
+    assert soco.play_mode == "NORMAL"
+
+    await hass.services.async_call(
+        MP_DOMAIN,
+        SERVICE_SHUFFLE_SET,
+        {
+            "entity_id": "media_player.zone_a",
+            "shuffle": True,
```

Comment:
```suggestion
            ATTR_ENTITY_ID: "media_player.zone_a",
            ATTR_SHUFFLE: True,
```
It would be really nice if we could use more constant throughout these tests

Reply:
Excellent point, I'll do a follow up pr.

Reply:
Oh I already have that opened now